packages/piro/test/Main_AnalysisDriver_Tpetra.cpp

```diff
@@ -128,11 +131,17 @@ int main(int argc, char *argv[]) {
 
 
       Stratimikos::DefaultLinearSolverBuilder linearSolverBuilder;
+
+#ifdef HAVE_PIRO_IFPACK2
       typedef Thyra::PreconditionerFactoryBase<double>              Base;
       typedef Thyra::Ifpack2PreconditionerFactory<Tpetra_CrsMatrix> Impl;
       linearSolverBuilder.setPreconditioningStrategyFactory(
           Teuchos::abstractFactoryStd<Base, Impl>(), "Ifpack2");
+#endif
+
+#ifdef HAVE_PIRO_MUELU
       Stratimikos::enableMueLu<int, int>(linearSolverBuilder);
```

Comment:
FYI, we plan to get rid of support for `GlobalOrdinal=int` at some point in the next few months.  Consider using the default Tpetra types here.  For example:
```c++
using LO = Tpetra::Map<>::local_ordinal_type;
using GO = Tpetra::Map<>::global_ordinal_type;
Stratimikos::enableMueLu<LO, GO>(linearSolverBuilder);
```